src/test/java/uk/sky/cqlmigrate/CqlMigratorImplTest.java

```diff
@@ -229,10 +236,10 @@ public void shouldThrowExceptionForBootstrapWithMissingSemiColon() throws Except
         Collection<Path> cqlPaths = singletonList(getResourcePath("cql_bootstrap_missing_semicolon"));
 
         //when
-        MIGRATOR.migrate(CASSANDRA_HOSTS, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
+        MIGRATOR.migrate(CASSANDRA_HOSTS, LOCAL_DC, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
 
         //then
-        cluster.connect(TEST_KEYSPACE);
+        //throws exception
```

Comment:
Could we catch the throwable and assert it's type instead of nothing? This has highlighted that it's not obvious which functional call the exception was previously being thrown from